test/types/add.ts

```diff
@@ -0,0 +1,8 @@
+import { expectType, expectError} from 'tsd';
+import * as R from '../../es/index';
+
+expectType<number>(R.add(2, 3)); // => 5
+expectType<number>(R.add(7)(10)); // => 17
```

Comment:
Since ramda apis are auto curried, we should include tests of partial applied `R.add` , such as:

```ts
expectType<(a: number) => number>(R.add(1));
```

Reply:
yeah idk about those -
i think if the `expectType<number>(R.add(7)(10));` it should be safe to assume currying is working correctly

testing partially applied functions would work but i'm not sure at what point it'd stop being tests and start being baselines - not that that's a bad thing

Reply:
but, well, if these *are* supposed to be baselines maybe we'd want to use those instead of tests...

... but actually i'm not sure if there's a module out there that generates baselines currently

Reply:
Ya I'm not sure what to do about this one.. I guess better to cover it even if it is a bit annoying